Network/HTTP/Req.hs

```diff
@@ -7,83 +7,83 @@
+-- level. The library also carefully hides underlying types from the lower-level
+-- @http-client@ package because those types are not safe enough (for example
+-- 'L.Request' is an instance of 'Data.String.IsString' and, if it's
 -- malformed, it will blow up at run-time).
 --
--- “Expandable” refers to the ability of the library to be expanded without
--- ugly hacking. For example, it's possible to define your own HTTP methods,
--- new ways to construct body of request, new authorization options, new
--- ways to actually perform request and how to represent\/parse response. As
--- user extends the library to satisfy his\/her special needs, the new
--- solutions work just like built-ins. That said, all common cases are
--- covered by the library out-of-the-box.
---
--- “High-level” means that there are less details to worry about. The
--- library is a result of my experiences as a Haskell consultant, working
--- for several clients who have very different projects and so the library
--- adapts easily to any particular style of writing Haskell applications.
--- For example, some people prefer throwing exceptions, while others are
--- concerned with purity: just define 'handleHttpException' accordingly when
--- making your monad instance of 'MonadHttp' and it will play seamlessly.
--- Finally, the library cuts boilerplate considerably and helps write
--- concise, easy to read and maintain code.
+-- “Expandable” refers to the ability to create new components for dealing with HTTP without
+-- having to resort to ugly hacking. For example, it's possible to define your own
+-- HTTP methods, create new ways to construct the body of a request, create new authorization options,
+-- perform a request in a different way, and create your own methods to parse and represent a response.
+-- As a user extends the library to satisfy his/her special needs, the new solutions will work
+-- just like the built-ins. However, all of the common cases are also covered by the library
+-- out-of-the-box.
+--
+-- “High-level” means that there are less details to worry about. The library
+-- is a result of my experiences as a Haskell consultant. Working for several
+-- clients, who had very different projects, showed me that the library should adapt easily to
+-- any particular style of writing Haskell applications. For example, some
+-- people prefer throwing exceptions, while others are concerned with purity.
+-- Just define `handleHttpException` accordingly when making your monad
+-- instance of `MonadHttp` and it will play together seamlessly. Finally, the library
+-- cuts boilerplate down considerably, and helps you write concise, easy to read, and
+-- maintainable code.
 --
 -- === Using with other libraries
 --
---     * You won't need low-level interface of @http-client@ most of the
---       time, but when you do, it's better import it qualified because it
+--     * You won't need the low-level interface of @http-client@ most of the
```

Comment:
I think "import it qualified" is better here because we refer to `import qualified MyModules as M` construct. "Import and qualify" conveys a different meaning. What do you think?